charts/keycloak/templates/statefulset.yaml

```diff
@@ -126,15 +126,15 @@ spec:
             {{- tpl . $ | nindent 12 }}
           {{- end }}
           livenessProbe:
```

Comment:
So do you want to get rid of the following parameters then?

```
keycloak.livenessProbe.initialDelaySeconds
keycloak.livenessProbe.timeoutSeconds
keycloak.readinessProbe.initialDelaySeconds
keycloak.readinessProbe.timeoutSeconds
```

I agree that having only the `livenessProbe/readinessProbe` parameters is probably cleaner solution, but this would break backward compatibility. For example, if someone currently have `keycloak.livenessProbe.timeoutSeconds` configured, then after upgrade to the new version this value will no longer be used (and so default will be applied).

Please let me know if this is acceptable and I'll apply the changes.

Reply:
Good point. I hadn't realized that. We'd need to bump the major version. The question is whether that warrants a major update. I tend to say yes because it's the cleaner solution. Would also need a note in the upgrading section of the readme. WDYT?

Reply:
Well, it would technically be a breaking change so I think it should be included in major version update. As for the impact (i.e. is it big enough to be considered a major change) I think it depends. It is definitely major for us, as right now this is the only remaining problem that prevents us from using this chart with Istio + mTLS. And I can see how it could be useful in other cases (like in #46), so maybe it is justified to be a major update.

Do you have any version number in mind? Should it be 6.0.0?

Reply:
It should be 6.0.0.

Reply:
I'll implement and test the changes once I get to my laptop (probably on Monday).